### PR TITLE
Add `path_presence_test` tag in metadata.proto.

### PR DESCRIPTION
**Pull request by wenovus**
This will mark tests that only test for path existence/validity such as https://github.com/openconfig/featureprofiles/tree/main/feature/gnmi/otg_tests/telemetry_basic_check_test so that tools (e.g. coverage tools) can have better insight on what the coverage means.

----

Emum is another choice for the data type but I'm not sure if it makes sense to have coverage types other than these two -- too fine grained seems too hard to track so I'd like to keep this as simple as possible.

### CODE REVIEW

**Review by dplore — DISMISSED** [review]

**Comment by wenovus** [comment]
@dplore Need another approval after merge.

**Review by dplore — APPROVED** [review]

**Pull request merged**
